app/_posts/2024-06-25-kuma-2-8-0.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kuma 2.8 release with MeshExternalService, MeshPassthrough and more...
+description: Kuma 2.8 release with MeshExternalService, MeshPassthrough and more...
+date: 2024-06-25
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.8, featuring new and exciting capabilities such as `MeshExternalService`, `MeshPassthrough` policy, along with numerous improvements and groundwork for future developments.
+
+# Notable features
+* A new `MeshExternalService` resource that overcomes existing limitations with the `ExternalService` resource.
+* A new `MeshPassthrough` policy that allows exposing external endpoints for rich clients through the mesh and support for wildcard domains.
+* HostnameGenerator provides a way to generate custom domains for your `MeshExternalServices`, offering enhanced security.
+
+Feel free to check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.8.0) for the full list of changes.
+
+## HostnameGenerator
+
+It's a brand new resource [`HostnameGenerator`](/docs/2.8.x/networking/hostnamegenerator) that enables you to generate custom domain for your `MeshExternalService`. By default Kuma is generating a domain for your `ExternalService` by adding suffix `.mesh` to the name of your `ExternalService`. With this release, you now have control over the domain.
+
+Let's take a look at this example:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: HostnameGenerator
+metadata:
+  name: example
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    meshExternalService:
+      matchLabels:
+        kuma.io/origin: zone
+  template: "{{ .DisplayName }}.svc.meshext.local"
+```
+
+and now let's create a `MeshExternalService`
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshExternalService
+metadata:
+  name: mes-http
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default
+spec:
+  match:
+    type: HostnameGenerator
+    port: 80
+    protocol: http
+  endpoints:
+  - address: httpbin.org
+    port: 80
+```
+
+In a brand new tab in the GUI, we can now see the generated domain address for our new `MeshExternalService`.
+
+![MeshExternalService view with a custom domain provided by the HostnameGenerator](/assets/images/blog/kuma_2_8_mes_hostname.png)
+
+## MeshExternalService
+The `ExternalService` resource struggled with various issues, leading us to develop the more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This brand-new resource enables you to add external endpoints to your mesh and overcome the limitations of the `ExternalService`. `MeshExternalService` will eventually replace `ExternalService` and is currently in the alpha phase.
```

Comment:
it uses "brand-new" once more and I don't think "to struggle" is the right verb when it comes to policies 🤔 

```suggestion
The `ExternalService` resource had some shortcomings, leading us to develop a more powerful [`MeshExternalService`](/docs/2.8.x/networking/meshexternalservice/) resource. This new resource enables you to add external endpoints to your mesh and overcome the limitations (should we name them? I feel like here we're just repeating what was said in the previous paragraph) of the `ExternalService`. `MeshExternalService` is currently in the alpha phase and will eventually replace `ExternalService`.
```